src/code-templates/libraries/logger/test/index.test.ts

```diff
@@ -7,77 +7,71 @@ beforeEach(() => {
 });
 
 describe('logger', () => {
-  test('When no explicit configuration is set, info logs are written', async () => {
-    // Arrange
-    const stdoutStub = sinon.stub(process.stdout, 'write');
+  let stdoutStub: sinon.SinonStubbedMember<typeof process['stdout']['write']>;
 
+  beforeEach(() => {
+    // Must stub the process.stdout before we configure the logger
+    // as it would fail when the underlying logger is pino, and we run in WebStorm
+    // See more here - practicajs/practica#225
+    stdoutStub = sinon.stub(process.stdout, 'write');
```

Comment:
Why not keep this within the tests? As a general rule of thumb, anything that directly affects the test is much more readable when its part of the test story (vs external side effect)

Reply:
Agree but duplicating this multiple times will make the test less readable in my opinion

Reply:
less readable as every time you will see a long comment which will make the test longer and people will just ignore it, instead, I spied in every function and added the comment only to the first stub